src/content/contributing/lighthouse/_index.en.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
@aswinsuryan please use make server locally in your laptop to verify the rendering, it's not working in a few places (see comments bellow).

Reply:
Oh I missed that step for this one, thanks for checking it. Will verify it.

Reply:
Fixed the issues.